src/main/kotlin/racingcar/domain/Location.kt

```diff
@@ -0,0 +1,22 @@
+package racingcar.domain
+
+data class Location(
+    val value: Int
```

Comment:
```suggestion
    val value: Int = 0
```
BASE_LOCATION을 사용하는 것도 좋으나, 기본값을 활용해보면 어떨까요?

Reply:
VO를 사용할 때 주로 많이 사용 되는 값은 미리 선언해두어 사용하는 방식을 많이 사용해왔었는데요! (ex. Money.Zero)
미리 선언해두는 대신 기본 값을 사용하게되면 얻는 이점은 어떤 부분이 있을까요? 🤔

Reply:
기본값을 사용하게 되면, BASE_LOCATION이라는 변수를 몰라도 `Location()` 으로 생성할 수 있어요.